modules/porous_flow/include/materials/PorousFlowMaterialPermeabilityFromPorosity.h

```diff
@@ -0,0 +1,76 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef POROUSFLOWMATERIALPERMEABILITYFROMPOROSITY_H
+#define POROUSFLOWMATERIALPERMEABILITYFROMPOROSITY_H
+
+#include "DerivativeMaterialInterface.h"
+#include "Material.h"
+
+#include "PorousFlowDictator.h"
+
+//Forward Declarations
+class PorousFlowMaterialPermeabilityFromPorosity;
```

Comment:
I suggest calling this PorousFlowPermeabilityKozenyCarman .  Other people might want other permeability-porosity relationships